test/functional/Java21Only/src/org/openj9/test/jep442/downcall/PrimitiveTypeTests1.java

```diff
@@ -21,22 +21,29 @@
  */
 package org.openj9.test.jep442.downcall;
 
-import org.testng.annotations.Test;
 import org.testng.Assert;
-import org.testng.AssertJUnit;
+import org.testng.annotations.Test;
 
 import java.io.UnsupportedEncodingException;
 import java.lang.invoke.MethodHandle;
 import java.util.Arrays;
 
 import java.lang.foreign.Arena;
-import java.lang.foreign.Linker;
 import java.lang.foreign.FunctionDescriptor;
+import java.lang.foreign.Linker;
 import java.lang.foreign.MemoryLayout;
```

Comment:
Please correct the order of imports and remove the unused import for `MemoryLayout`.

Reply:
> Please correct the order of imports

I don't see there is any misplaced imports (please point it out if I ignored).

> remove the unused import for MemoryLayout

Fixed.